src/modules/dotnet-test.test.ts

```diff
@@ -55,6 +56,55 @@ describe("DotnetTest", () => {
     });
 
     // #endregion runBySolution
+
+    // -----------------------------------------------------------------------------------------
+    // #region watchMode
+    // -----------------------------------------------------------------------------------------
+
+    describe("watchMode", () => {
+        const defaultArgs = ["test", "--no-build", "--no-restore"];
+        const spawnSyncSpy = jest.spyOn(child_process, "spawnSync");
+
+        beforeEach(() => {
+            spawnSyncSpy.mockReset();
```

Comment:
Is this `mockReset` necessary? Haven't seen it needed anywhere else in the codebase - I think `restoreMocks: true` in `jest.config.js` might handle clearing out mocks automatically.

Reply:
Probably not. 👍🏻